.gitignore

```diff
@@ -104,4 +104,7 @@ dist
 .tern-port
 
 # .env
-*.env
\ No newline at end of file
+*.env
+
+#package lock, will be generated anyway on install
+package-lock.json
```

Comment:
Lock file should not be ignored

Reply:
sory i wasn't aware my commits went to this pull, but like why should it not be ignored? it gets generated from npm anyway, so having it there has no effect

Reply:
https://docs.npmjs.com/cli/v7/configuring-npm/package-lock-json
Maybe try searching before asking next time